doc/cabal-package.rst

```diff
@@ -1188,14 +1187,14 @@ Test suites
 The test suite may be described using the following fields, as well as
 build information fields (see the section on `build information`_).
 
-.. pkg-field:: type: interface (required)
+.. pkg-field:: type: interface
 
     The interface type and version of the test suite. Cabal supports two
-    test suite interfaces, called ``exitcode-stdio-1.0`` and
+    test suite interfaces, called ``exitcode-stdio-1.0`` (default) and
```

Comment:
The documentation needs to say when it became the default (and thus optional).  E.g. "default since cabal-version 3.8" or so.